app/models/moirai/translation.rb

```diff
@@ -3,5 +3,15 @@
 module Moirai
   class Translation < Moirai::ApplicationRecord
     validates_presence_of :key, :locale
+
+    def find_file_path
+      @key_finder ||= KeyFinder.new
+      @key_finder.file_path_for(key, locale: locale)
+    end
```

Comment:
maybe the right place where this belongs to is the KeyFinder. So you are sure to not instantiate too many